WordPress/src/main/java/org/wordpress/android/util/FluxCUtils.java

```diff
@@ -122,14 +122,20 @@ public static MediaModel mediaModelFromLocalUri(@NonNull Context context,
             }
         }
 
+        String title = filename;
+        // Remove extension from title
+        if (fileExtension != null && title.contains("." + fileExtension)) {
+            title = title.substring(0, title.lastIndexOf(fileExtension) - 1);
+        }
+
```

Comment:
If I'm not wrong, filename (and so title) could be null; maybe we should null check on title. wdyt?

Reply:
Good point!

Although filename doesn't seem to be set to null as in a previous line it checks if `!file.exists()` which means it should be a valid file location from where the file name is derived. But there's no harm adding a null check 😄.

I'm setting file name to an empty string if null in fc052ec so that we don't set a null filename as well. 

Ready for another look 🙇

Reply:
Ah yep, I see what you mean 👍 ! And agree with you no harm in adding the check 😄 .